app/src/main/res/layout-land/activity_edit.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textAppearance="?android:attr/textAppearanceMedium"
+        android:text="@string/StartTimeLabel"
+        android:id="@+id/StartTimeLabel"
+        android:layout_alignParentTop="true"
+        android:layout_alignParentLeft="true"
+        android:layout_centerHorizontal="true"
+        android:layout_toLeftOf="@+id/StartDatePicker"
+        android:layout_alignTop="@+id/StartTimePicker"
+        android:layout_alignBottom="@+id/StartTimePicker"/>
+
+    <DatePicker
```

Comment:
Picker werden, wegen zu verschwenderischen Platzverbrauch, nicht in normalen layouts genutzt, sondern nur in den Dialogen.